### PR TITLE
Updates prometheus version to 2.1.0 (current latest)

### PR DESCRIPTION
**Pull request by jasongwartz**

### CODE REVIEW

**Comment by alexellis** [comment]
Hey @jasongwartz thanks for this, but it should be in a new versioned folder, we don't want to break compatibility for people using the image / Dockerfile. Can you do that please and state how you've tested it too 👍

**Comment by jasongwartz** [comment]
Yup, very good point! I've done that and pushed changes now, numbering both prometheus directories.

I've built this on a Raspberry Pi 3, tested it in a Kubernetes cluster of Pi 2 and 3s. Works exactly as your old image did, but now with the new tsdb and the other improvements of Prometheus v2.

File: images/armhf/prometheus-2.1.0/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,28 @@
+FROM armhf/alpine:3.5
```
Comment (alexellis) [inline images/armhf/prometheus-2.1.0/Dockerfile]: This should just be "alpine:3.6" now.

**Comment by jasongwartz** [comment]
Out of date now

**Comment by alexellis** [comment]
This has been moved to the contrib folder of openfaas/faas with the latest versions.

**Pull request closed without merging**